operations/customer-success/README.md

```diff
@@ -1,5 +1,27 @@
 # Customer Success
 
+## Vision
+
+Deliver the ideal customer journey experience to all customers at scale.
+
+### Customer Success Journey Principles
+
+- See first from the customer’s perspective (golden rule)
+- Strive to be a partner, looking to meet both the current and future needs of the customer
+- Be solution-oriented
+- Focus on customer outcomes and creating value
+- Make Mattermost better by sharing learnings and feedback
+
+### Customer Segmentation
+
+#### Vision
```

Comment:
0/5: Wondering if this may be extraneous - the points below are very clear and vision-focused.

Reply:
@justinegeffen Apologies, not sure if I understand.  Are you saying to remove the points?

Reply:
Sorry! I meant the word "Vision". :)

Reply:
@justinegeffen  This looks great!